test/API/CMakeLists.txt

```diff
@@ -9,20 +9,6 @@
 # help@hdfgroup.org.
 #
 
-#------------------------------------------------------------------------------
-# Set module path
-#------------------------------------------------------------------------------
-set(HDF5_TEST_API_CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/CMake")
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${HDF5_TEST_API_CMAKE_MODULE_PATH})
-
-# TODO: probably not necessary
-#------------------------------------------------------------------------------
-# Setup CMake Environment
-#------------------------------------------------------------------------------
-if (WIN32)
-  message("The HDF5 API test suite is currently not supported on this platform." FATAL_ERROR)
```

Comment:
There shouldn't be any reason that these tests can't be built and run on Windows with the native VOL connector. Support for other VOL connectors on Windows is a different matter.

Reply:
if (HDF5_TEST_PARALLEL)
  set (HDF5_TEST_API_HAVE_PARALLEL 1)
endif ()

I can't find any other us of this variable.

Reply:
Ah yes, that should have been removed when I switched over to checking for H5_HAVE_PARALLEL instead.